dhall/mainnet.dhall

```diff
@@ -0,0 +1,9 @@
+{ name         = "mainnet"
+, keyPrefix    = "mainnet_wallet"
+, relays       = "relays.cardano-mainnet.iohk.io"
+, updateServer = "https://update-cardano-mainnet.iohk.io"
+, reportServer = "http://report-server.cardano-mainnet.iohk.io:8080"
+, installDirectorySuffix = ""
+, macPackageSuffix       = ""
+, walletPort             = 8090
```

Comment:
For example, `walletPort` belongs in a wallet specific configuration, as does `reportServer` and possibly some others. My idea is to have the _base configuration_ and the have the _per feature configuration_ (so for wallet, we can have _walletConfig.dhall_).